#163518685 Mock SendGrid Mail

abayo-luc opened the pull request:
#### What does this PR do?
Disable sending reset password email from test
#### Description of Task to be completed?
```POST /api/users/reset_password```
#### How should this be manually tested?
-Clone the repo and check out `bg-moch-sendgrid-mail-163518685` branch
- Go under ```helpers``` directories, and change dummyUser email to your own working email.
- Run ```yarn test```, and check your email, you will see nothing. 
#### Any background context you want to provide?
before running anything, ensure packages are updated ```yarn``` 
#### What are the relevant pivotal tracker stories?
163518685

abayo-luc committed:
#163518685 Use nock for mokcing sendgrid email

abayo-luc committed:
#163518685 Use nock for mokcing sendgrid email

abayo-luc committed:
#163518685 Use nock for mokcing sendgrid email

abayo-luc committed:
#163518685 Use nock for mokcing sendgrid email

niomwungeri-fabrice reviewed (approved):
Good Job mate!

espoirMur commented on tests/users/passwordReset.test.js:
```diff
@@ -1,21 +1,26 @@
-        ...dummyUser2
+        ...dummyUser
       });
+    nock("https://api.sendgrid.com")
```
Nice job

abayo-luc committed:
Merge branch 'develop' into bg-moch-sendgrid-mail-163518685

abayo-luc committed:
Update passwordReset.test.js

abayo-luc committed:
Update passwordReset.test.js

espoirMur commented on helpers/data.js:
```diff
@@ -10,3 +10,49 @@ export const data = {
     username: "jean786786"
   }
 };
+
```
Can you please move this file into test/helpers/ as the sendGridResponse object is a test data?

Musigwa committed:
#163518689 Create the new article (#19)

* feat(article): Create the new article with tags

- ensure the endpoint is protected by the jwt
- implement the create article request handler
- ensure that every tag put to the article is in the database

* [Feature #163518689] Add the tags list to response

* [Feature #163518689] Fix the rebase conflicts

* [Feature #163518689] Solve rebasing conflicts

abayo-luc committed:
16407007 Folder restructuring (#28)

abayo-luc committed:
Rebase develop and fix conflict

abayo-luc committed:
Rebase develop and fix conflict

abayo-luc committed:
Rebase develop and fix conflict

abayo-luc committed:
Rebase develop and fix conflict

abayo-luc committed:
Rebase develop and fix conflict

YvesIraguha committed:
#163518699 Calculate the Time Length for Reading an Article  (#30)

* [feature #163518699] calculate readtime functionality

* [feature #163518699] Add the function for calculating the readtime

abayo-luc committed:
Rebase develop

abayo-luc committed:
Fix conflict

puritybirir commented on models/Article.js:
```diff
@@ -11,9 +11,26 @@ export default (sequelize, DataTypes) => {
-    { tableName: "articles" }
+    {
+      tableName: "articles"
+      // hooks: {
```
Why are these comments going to staging?

abayo-luc replied:
@puritybirir that one was from develop, but now I removed it.

puritybirir commented:
Name of the Branch should also be bg-mock-sendgrid and not bg-moch-sendgrid

abayo-luc committed:
Update Article.js

abayo-luc commented:
Duplicate of #31
